organization_token_integration_test.go

```diff
@@ -20,24 +21,41 @@ func TestOrganizationTokensCreate(t *testing.T) {
 
 	var tkToken string
 	t.Run("with valid options", func(t *testing.T) {
-		ot, err := client.OrganizationTokens.Create(ctx, orgTest.Name)
+		ot, err := client.OrganizationTokens.Create(ctx, orgTest.Name, OrganizationTokenCreateOptions{})
 		require.NoError(t, err)
 		require.NotEmpty(t, ot.Token)
 		tkToken = ot.Token
 	})
 
 	t.Run("when a token already exists", func(t *testing.T) {
-		ot, err := client.OrganizationTokens.Create(ctx, orgTest.Name)
+		ot, err := client.OrganizationTokens.Create(ctx, orgTest.Name, OrganizationTokenCreateOptions{})
 		require.NoError(t, err)
 		require.NotEmpty(t, ot.Token)
 		assert.NotEqual(t, tkToken, ot.Token)
 	})
 
 	t.Run("without valid organization", func(t *testing.T) {
-		ot, err := client.OrganizationTokens.Create(ctx, badIdentifier)
+		ot, err := client.OrganizationTokens.Create(ctx, badIdentifier, OrganizationTokenCreateOptions{})
 		assert.Nil(t, ot)
 		assert.EqualError(t, err, ErrInvalidOrg.Error())
 	})
+
+	t.Run("without an expiration date", func(t *testing.T) {
+		ot, err := client.OrganizationTokens.Create(ctx, orgTest.Name, OrganizationTokenCreateOptions{})
+		require.NoError(t, err)
+		require.NotEmpty(t, ot.Token)
+		tkToken = ot.Token
```

Comment:
suggestion: We could check in this test (and the ones for user and team tokens) that the expiry actually ends up being what we expect it to be